Makefile

```diff
@@ -1,7 +1,8 @@
+# Y - in response to Would you like to add the SemaphoreCI addon?
 # Y - in response to Would you like to add the Nginx addon?
 # Y - in response to Would you like to add the PhraseApp addon?
 # Y - in response to Would you like to add the Devise addon?
-common_addon_prompts = Y\nY\nY\n
+common_addon_prompts = N\nY\nY\nY\n
```

Comment:
Should we change it to `Y` @longnd ?

Reply:
definitely yes, I was too hurry when making the change 🙈. Fixed in 421d5a6